### PR TITLE
[5.2] Translation for Publish & Feature

### PR DESCRIPTION
**Pull request by Kostelano**
Pull Request for Issue # .

### Summary of Changes
Set a language that is clearly different from English. Go to the **Users --- Manage** menu, in the **User Groups** column, select **Permissions**. A little lower in the list there will be a line Publish & Feature (com_content.transition.7) without translation.

The same on the **Users: Groups ---> Permissions** page.

Screenshot_1

PR fixes this small problem, now we see the line translated.

**UPD**
Added files to update translation strings for `#__assets`.
Test by installing the UPDATE package on your 5.2 test sites.

### CODE REVIEW

**Comment by brianteeman** [comment]
Good spot. This was missed from [4.0] Make transitions translateable #35457

You will need to add an update sql for this as well. Check that PR and you will see what is needed

File: administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+UPDATE `#__assets` SET `title` = 'PUBLISH' WHERE `title`= 'Publish';
+UPDATE `#__assets` SET `title` = 'UNPUBLISH' WHERE `title`= 'Unpublish';
+UPDATE `#__assets` SET `title` = 'TRASH' WHERE `title`= 'Trash';
+UPDATE `#__assets` SET `title` = 'ARCHIVE' WHERE `title`= 'Archive';
+UPDATE `#__assets` SET `title` = 'FEATURE' WHERE `title`= 'Feature';
+UPDATE `#__assets` SET `title` = 'UNFEATURE' WHERE `title`= 'Unfeature';
+UPDATE `#__assets` SET `title` = 'PUBLISH_AND_FEATURE' WHERE `title`= 'Publish & Feature';
```
Comment (richard67) [inline administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql]: Maybe we should also add the asset names to the where conditions as the titles might also be used by 3rd party extensions and we should not update these.

Comment (Kostelano) [reply administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql]: Please help me make this right by changing the line here

**Comment by richard67** [comment]
@Hackwar Does this count as new feature so it should go into 5.3-dev? Or is it ok for 5.2-dev?

File: administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql (hunk 1 above)
Comment (richard67) [reply administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql]:
```suggestion
UPDATE `#__assets` SET `title` = 'PUBLISH_AND_FEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Publish & Feature';
```
Do the same for all other update statements here and in the PostgreSQL file (for PostgreSQL use of course other names quotes `"`)
The additional condition added with my suggestion can be the same for all these update statements.

Comment (richard67) [reply administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql]: Hmm, I just see the other, old PR #35457 for 4.0.3 did not do that either. But I suggest to do it here so we are safer.

Comment (Kostelano) [reply administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql]: Thank you, I made the changes

File: administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+UPDATE `#__assets` SET `title` = 'PUBLISH' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Publish';
+UPDATE `#__assets` SET `title` = 'UNPUBLISH' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Unpublish';
+UPDATE `#__assets` SET `title` = 'TRASH' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Trash';
+UPDATE `#__assets` SET `title` = 'ARCHIVE' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Archive';
+UPDATE `#__assets` SET `title` = 'FEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Feature';
+UPDATE `#__assets` SET `title` = 'UNFEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Unfeature';
+UPDATE `#__assets` SET `title` = 'PUBLISH_AND_FEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title`= 'Publish & Feature';
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-09-24.sql]:
```suggestion
UPDATE `#__assets` SET `title` = 'PUBLISH' WHERE `name` LIKE 'com_content.transition.%' AND `title` = 'Publish';
UPDATE `#__assets` SET `title` = 'UNPUBLISH' WHERE `name` LIKE 'com_content.transition.%' AND `title` = 'Unpublish';
UPDATE `#__assets` SET `title` = 'TRASH' WHERE `name` LIKE 'com_content.transition.%' AND `title` = 'Trash';
UPDATE `#__assets` SET `title` = 'ARCHIVE' WHERE `name` LIKE 'com_content.transition.%' AND `title` = 'Archive';
UPDATE `#__assets` SET `title` = 'FEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title` = 'Feature';
UPDATE `#__assets` SET `title` = 'UNFEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title` = 'Unfeature';
UPDATE `#__assets` SET `title` = 'PUBLISH_AND_FEATURE' WHERE `name` LIKE 'com_content.transition.%' AND `title` = 'Publish & Feature';
```

File: administrator/components/com_admin/sql/updates/postgresql/5.2.0-2024-09-24.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+UPDATE "#__assets" SET "title" = 'PUBLISH' WHERE "name" LIKE 'com_content.transition.%' AND "title"= 'Publish';
+UPDATE "#__assets" SET "title" = 'UNPUBLISH' WHERE "name" LIKE 'com_content.transition.%' AND "title"= 'Unpublish';
+UPDATE "#__assets" SET "title" = 'TRASH' WHERE "name" LIKE 'com_content.transition.%' AND "title"= 'Trash';
+UPDATE "#__assets" SET "title" = 'ARCHIVE' WHERE "name" LIKE 'com_content.transition.%' AND "title"= 'Archive';
+UPDATE "#__assets" SET "title" = 'FEATURE' WHERE "name" LIKE 'com_content.transition.%' AND "title"= 'Feature';
+UPDATE "#__assets" SET "title" = 'UNFEATURE' WHERE "name" LIKE 'com_content.transition.%' AND "title"= 'Unfeature';
+UPDATE "#__assets" SET "title" = 'PUBLISH_AND_FEATURE' WHERE "name" LIKE 'com_content.transition.%' AND "title"= 'Publish & Feature';
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/postgresql/5.2.0-2024-09-24.sql]:
```suggestion
UPDATE "#__assets" SET "title" = 'PUBLISH' WHERE "name" LIKE 'com_content.transition.%' AND "title" = 'Publish';
UPDATE "#__assets" SET "title" = 'UNPUBLISH' WHERE "name" LIKE 'com_content.transition.%' AND "title" = 'Unpublish';
UPDATE "#__assets" SET "title" = 'TRASH' WHERE "name" LIKE 'com_content.transition.%' AND "title" = 'Trash';
UPDATE "#__assets" SET "title" = 'ARCHIVE' WHERE "name" LIKE 'com_content.transition.%' AND "title" = 'Archive';
UPDATE "#__assets" SET "title" = 'FEATURE' WHERE "name" LIKE 'com_content.transition.%' AND "title" = 'Feature';
UPDATE "#__assets" SET "title" = 'UNFEATURE' WHERE "name" LIKE 'com_content.transition.%' AND "title" = 'Unfeature';
UPDATE "#__assets" SET "title" = 'PUBLISH_AND_FEATURE' WHERE "name" LIKE 'com_content.transition.%' AND "title" = 'Publish & Feature';
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 048f044f7e00091efccee3e6bc3bc1edf1d32f39<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44133">issues.joomla.org/tracker/joomla-cms/44133</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 048f044f7e00091efccee3e6bc3bc1edf1d32f39<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44133">issues.joomla.org/tracker/joomla-cms/44133</a>.</sub>

**Comment by alikon** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44133">issues.joomla.org/tracker/joomla-cms/44133</a>.</sub>

**Comment by pe7er** [comment]
Thanks @Kostelano !

**Comment by richard67** [comment]
@pe7er You should have checked the names of the update SQL scripts added by this PR before merging. The PR was merged after the release of 5.2.0, so they should have been renamed from "5.2.0-2024-09-24.sql" to "5.2.1-2024-09-24.sql". I've made a PR to fix that: #44388 .

**Pull request merged**